tensorflow/models/rnn/ptb/ptb_word_lm.py

```diff
@@ -245,13 +245,24 @@ def run_epoch(session, m, data, eval_op, verbose=False):
   start_time = time.time()
   costs = 0.0
   iters = 0
-  state = m.initial_state.eval()
+  state = []
+  for c, h in m.initial_state:
+    state.append((c.eval(), h.eval()))
   for step, (x, y) in enumerate(reader.ptb_iterator(data, m.batch_size,
                                                     m.num_steps)):
-    cost, state, _ = session.run([m.cost, m.final_state, eval_op],
-                                 {m.input_data: x,
-                                  m.targets: y,
-                                  m.initial_state: state})
+    fetches = []
+    fetches.append(m.cost)
+    fetches.append(eval_op)
+    for c, h in m.final_state:
+        fetches.append(c)
+        fetches.append(h)
+    feed_dict = {}
+    feed_dict[m.input_data] = x
+    feed_dict[m.targets] = y
+    for i, (c, h) in enumerate(m.initial_state):
+        feed_dict[c], feed_dict[h] = state[i]
+    cost, _, *states = session.run(fetches, feed_dict)
```

Comment:
My text editor inserted 4 spaces for indent instead of 2 spaces...
Updated the codes, and checked that it passes the sanity checks in my local machine.